src/utils/settings.cpp

```diff
@@ -173,6 +173,11 @@ QString mp::Settings::get_daemon_settings_file_path() // temporary
     return file_for(daemon_root);
 }
 
+QString mp::Settings::get_client_settings_file_path() // idem
+{
+    return file_for("client");
```

Comment:
Nitpick: `"client"` looks like a candidate for `const auto client_root`?

Reply:
Right, I thought of that, but then if I do that, I should change any other places that use the `client` string and that's unrelated to this PR.  If you're fine with that, then I'll change to a `constexpr`.

Reply:
Yeah, let's :)

Reply:
Done.